tests/create-course.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test } from '@playwright/test';
+import { LoginPage, CoursePage } from '../src/pages/admin-ui';
+
+test('create new course', async ({ page }) => {
+    let loginPage = new LoginPage(page);
+    await loginPage.goto(process.env.HOBBYFARM_ADMIN_UI_URL as string);
+    let homePage = await loginPage.fillCredentialsAndSubmit(process.env.HOBBYFARM_ADMIN_UI_USR as string, process.env.HOBBYFARM_ADMIN_UI_PWD as string);
+    let contentManagementPage = await homePage.openContentManagementPage();
+    let coursePage = await contentManagementPage.openNewScenarioModalAndSave('testname' as string, 'testdescription' as string, '15' as string, '2' as string);
```

Comment:
Small typo, should be openNewCourseModalAndSave
This method needs to be created in ContentManagementPage (and call methods in CoursePage)